buildroot-external/patches/occu/0154-WebUI-DynamicResizing.patch

```diff
@@ -113,6 +113,138 @@
        gotoLoginPage = function() {
          location.href = "/login.htm";
        };
+--- occu/WebUI/www/rega/pages/tabs/control/devices.htm.orig
++++ occu/WebUI/www/rega/pages/tabs/control/devices.htm
+@@ -13,14 +13,9 @@
+   arItems = new Array();
+ </script>
+ 
+-<table width="100%" class="CLASS03900" cellpadding="0" cellspacing="0">
+-  <colgroup>
+-    <col style="width:20%;"/>
+-    <col style="width:80%;"/>
+-  </colgroup>
+-  <tr>
++<div id="devicepage" class="CLASS03900" style="width:100%">
+     <!-- Linke Spalte -->
+-    <td id="contentLeft" class="hidden">
++    <div id="contentLeft" class="hidden" style="width:20%;float:left;overflow:auto;">
+       <table class="filterTable CLASS03901 j_translate" cellpadding="0" cellspacing="0">
+         <tr>
+           <td style="vertical-align: top; border: 0px">
+@@ -120,10 +115,10 @@
+           </td>
+        </tr> 
+      </table>
+-   </td>
++   </div>
+ 
+     <!-- Rechte Spalte -->
+-    <td id="contentRight">
++    <div id="contentRight">
+       <div id="divRTD" class="CLASS03908">
+         <table id="tblRightContent" class="CLASS03909" cellpadding="0" cellspacing="0">
+          <tr class="CLASS03910">
+@@ -131,9 +126,8 @@
+          </tr>
+         </table>
+       </div>
+-    </td>
+-  </tr>
+-</table>
++    </div>
++</div>
+ <script type="text/javascript">
+        
+   loadChannels = function(id, specialFlag)
+--- occu/WebUI/www/rega/pages/tabs/control/functions.htm.orig
++++ occu/WebUI/www/rega/pages/tabs/control/functions.htm
+@@ -13,14 +13,9 @@
+   arItems = new Array();
+ </script>
+ 
+-<table width="100%" class="CLASS04000" cellpadding="0" cellspacing="0">
+-  <colgroup>
+-    <col style="width:20%;"/>
+-    <col style="width:80%;"/>
+-  </colgroup>
+-  <tr>
++<div id="functionpage" class="CLASS04000" style="width:100%">

```

Comment:
Also here style should be move to `style.css`.

Reply:
done